### PR TITLE
Calculate the area of a rectangle

### LINKED ISSUE #1: Create a function to calculate the surface area of a rectangle
You should have a function that takes two double parameters.
The function should take the two numbers and calculate the surface area and return the result as a double.

Area of a rectangle is Length x Width = Area.

### PR DESCRIPTION
**Pull request by EvanGGreen**
Added a function to the main class to calculate the area of a rectangle.

Added a call to print out the area of a rectangle.

### CODE REVIEW

File: HelloWorld/HelloWorld/Program.cs [hunk 1]
```diff
@@ -18,6 +18,9 @@ public double Calculate(double radis)
         return Pi * radis * radis;
     }
 
+    public double  calculate(double lenth, double width)
+    { return lenth * width; }
```
Comment (chrisggreen) [inline HelloWorld/HelloWorld/Program.cs]: This function name is the same as another function in this class.

Please use a name for the function that is more descriptive.  What is the function calculating?

**Review by chrisggreen — CHANGES_REQUESTED** [review]

File: HelloWorld/HelloWorld/Program.cs (hunk 1 above)
Comment (EvanGGreen) [reply HelloWorld/HelloWorld/Program.cs]: I have mended the name of the function. 

Has the fix been made

File: HelloWorld/HelloWorld/Program.cs [hunk 2]
```diff
@@ -18,6 +18,9 @@ public double Calculate(double radis)
         return Pi * radis * radis;
     }
 
+    public double  calculaterect1(double lenth, double width)
```
Comment (chrisggreen) [inline HelloWorld/HelloWorld/Program.cs]: Why are you putting a 1 on the function name?

You should also use title case to name a function.
IE: public double CalculateValue(double incomingValue).

Please refer to this link for information on standard naming practices.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

**Review by chrisggreen — CHANGES_REQUESTED** [review]

File: HelloWorld/HelloWorld/Program.cs (hunk 2 above)
Comment (EvanGGreen) [reply HelloWorld/HelloWorld/Program.cs]: What would you suggest I name this function to maintain clarity?

Comment (chrisggreen) [reply HelloWorld/HelloWorld/Program.cs]:
```suggestion
    public double  CalculateAreaOfRectangle(double length, double width)
```

**Pull request merged**